includes/rest-api/CMB2_REST.php

```diff
@@ -117,6 +124,10 @@ public static function maybe_init_and_hookup( CMB2 $cmb ) {
 	public function __construct( CMB2 $cmb ) {
 		$this->cmb = $cmb;
 		self::$boxes[ $cmb->cmb_id ] = $this;
+		self::$rest_slug = 'cmb2';
+		if( !empty( $this->cmb->prop( 'rest_slug_group' ) ) ) {
+            		self::$rest_slug = $this->cmb->prop( 'rest_slug_group' );
```

Comment:
@Mte90 Please don't mix tabs and spaces on the same line. Please switch to WordPress Coding Standards - using tabs only in PHP.

Reply:
Also, I'm not sure that the `prop` method sanitizes the value. Please check that and if not - would be nice to have something like `sanitize_key()` here.

Reply:
Nah, we don't need to sanitize this. The reason is that CMB2 config is meant to be coded (not added via user input/db), so if we needed to sanitize this, we would need to sanitize a bunch of other stuff in CMB2 that we don't (like field names/ids, etc).